core/meta/inc/TClass.h

```diff
@@ -102,8 +102,7 @@ friend class TStreamerInfo;
       /* had kIsEmulation = BIT(19), // Deprecated */
       kStartWithTObject = BIT(20),  // see comments for IsStartingWithTObject()
       kWarned      = BIT(21),
-      kHasNameMapNode = BIT(22),
-      kHasCustomStreamerMember = BIT(23) // The class has a Streamer method and it is implemented by the user or an older (not StreamerInfo based) automatic streamer.
```

Comment:
AFAICT there are still plenty of uses left that are only changed in the next commit; is the removal intentional here? If not, can you please make intermediate commits compile which will greatly help future bisection runs?

Reply:
Yes, it only semi-intentional for development purposes (where we do want the build failure) but still slip into the wrong commit. I will fix this.

On a related note, I most often bisect with a script (called by `bisect run script` similar to:
```
#!/bin/sh

(cd $ROOTSYS; ninja)  || exit 125

run_the_test_and_make_sure_the_exit_code_is_last
```
which allows to skip over the 'broken' build.

Reply:
Yes, I know this but it still gets problematic if you have to skip over too many commits. We had this recently that we could not bisect into a quite substantial PR and were left with something like 20 commits that could have introduced the bug. That wouldn't be the case here, but IMHO still a matter of good style to have all commits buildable (and ideally passing all tests)

Reply:
Indeed.  That's why I updated the commit :)